src/vs/base/browser/ui/list/listWidget.ts

```diff
@@ -712,6 +715,7 @@ export class List<T> implements ISpliceable<T>, IDisposable {
 	setFocus(indexes: number[]): void {
 		indexes = indexes.sort(numericSort);
 		this.focus.set(indexes);
+		this.view.domNode.setAttribute('aria-posinset', indexes.length ? (indexes[0] + 1).toString() : undefined);
```

Comment:
In that case I will jost not set it but remove the attribute, I believe that is the right thing to do